eZ/Publish/Core/Persistence/Cache/Tests/ContentHandlerTest.php

```diff
@@ -303,30 +305,68 @@ public function providerForCachedLoadMethodsMiss(): array
         ];
     }
 
+    /**
+     * @covers \eZ\Publish\Core\Persistence\Cache\ContentHandler::updateContent
+     */
+    public function testUpdateContent()
+    {
+        $this->loggerMock->expects($this->once())->method('logCall');
+
+        $innerContentHandlerMock = $this->createMock(SPIContentHandler::class);
+        $innerLocationHandlerMock = $this->createMock(SPILocationHandler::class);
+
+        $this->prepareHandlerMocks(
+            $innerContentHandlerMock,
+            $innerLocationHandlerMock,
+            1,
+            1,
+            0
+        );
+
+        $innerContentHandlerMock
+            ->expects($this->once())
+            ->method('updateContent')
+            ->with(2, 1, new UpdateStruct())
+            ->willReturn(new Content());
+
+        $this->cacheIdentifierGeneratorMock
+            ->expects($this->exactly(5))
+            ->method('generateTag')
+            ->withConsecutive(
```

Comment:
`withConsecutive` and `willReturnOnConsecutiveCalls` are deprecated in the newer versions of PHPUnit. Please consider changing those to `returnValueMap` here as it would be needed on merge-up anyway. For example, you can refer to https://github.com/ibexa/product-catalog/pull/1034/files#diff-df82e5aba4361380977da9b3843954e8614ebbb83c317a6bd45d4b53c0fe55f2R95.